### PR TITLE
Cleanup event

### PR DESCRIPTION
**Pull request by minaorangina**
@heron2014 So still WIP, but now when you click on a feed item to view an event, you can access all 4 types of event view (Confirmed event for a host, confirmed event for an invitee, poll for host and poll for invitee).

The pages render without displaying the data.  Will need rewriting at some stage

Also I've added `rsvps` to the event object, which in the original app was an object like this:
```js
{
  going: [],
  maybe: [],
  not_going: []
}
```
The arrays had user data objects inside them, representing who was going/not going etc

### CODE REVIEW

File: src/js/components/event/confirmed-event.js [hunk 1]
```diff
@@ -3,97 +3,70 @@
...
 import { View, Image, Text } from 'react-native';
-import RSVPsArea from './confirmed-event/RSVPs-area';
+import rsvpsArea from './confirmed-event/rsvps-area';
 import { eventNote } from '../../lib/confirmed-event-helpers';
 import formatDate from '../../lib/formatDate';
 import styles from '../../../styles';
 
 
-const ConfirmedEvent = ({ event, event_id, RSVPs, invitees, userIsHost,
+<<<<<<< HEAD
```
Comment (heron2014) [inline src/js/components/event/confirmed-event.js]: @minaorangina I think you missed clearing up the merge conflict here.

File: src/js/components/event/confirmed-event.js [hunk 2]
```diff
@@ -3,97 +3,70 @@
 /* eslint-disable no-unused-vars */
 import React from 'react';
 import { View, Image, Text } from 'react-native';
-import RSVPsArea from './confirmed-event/RSVPs-area';
+import rsvpsArea from './confirmed-event/rsvps-area';
```
Comment (heron2014) [inline src/js/components/event/confirmed-event.js]: @minaorangina here I think you missed to change the file name. The path is does not match this file name.

**Comment by heron2014** [comment]
I fixed these small issues. Have a look and poke me again if all good. I think is now ready to merge. Good work :)

File: src/js/components/event/confirmed-event.js (hunk 1 above)
Comment (heron2014) [inline src/js/components/event/confirmed-event.js]: I fixed it

File: src/js/components/event/confirmed-event.js (hunk 2 above)
Comment (heron2014) [inline src/js/components/event/confirmed-event.js]: I fixed it

File: src/js/components/event/confirmed-event.old.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
+/* eslint-disable no-shadow */
+/* eslint-disable no-else-return */
+/* eslint-disable no-unused-vars */
```
Comment (heron2014) [inline src/js/components/event/confirmed-event.old.js]: @minaorangina you left this old file as a reference?

Comment (minaorangina) [inline src/js/components/event/confirmed-event.old.js]: @heron2014 yes, just to help with the rebuilding

**Pull request merged**